playbook/app/pb_kits/playbook/pb_map/_map.tsx

```diff
@@ -1,16 +1,24 @@
 import React from 'react'
 import classnames from 'classnames'
 import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
-import { globalProps } from '../utilities/globalProps'
+import { globalProps, GlobalProps } from '../utilities/globalProps'
 
+import Flex  from "../pb_flex/_flex"
+import Icon from '../pb_icon/_icon'
+import Button from '../pb_button/_button'
```

Comment:
a nice way to shorten this (for future reference) is:
`import { Button, Flex, Icon} from "../../"`